sdk/communication/Azure.Communication.Chat/src/ChatClient.cs

```diff
@@ -281,8 +281,9 @@ public virtual Response DeleteChatThread(string threadId, CancellationToken canc
 
         private static HttpPipeline CreatePipelineFromOptions(ChatClientOptions options, CommunicationUserCredential communicationUserCredential)
         {
-            var httpPipelinePolicy = new CommunicationUserAuthenticationPolicy(communicationUserCredential);
-            HttpPipeline httpPipeline = HttpPipelineBuilder.Build(options, httpPipelinePolicy);
+            var token = new CommunicationTokenCredential(communicationUserCredential);
```

Comment:
nit: `token` could be renamed to `tokenCredential` to avoid being mistaken with one specific token object